portfolio/src/main/webapp/script.js

```diff
@@ -41,4 +41,21 @@ function handleResponse(response) {
 function addToDOM(textResponse) {
   const textContainer = document.getElementById('introduction-container');
   textContainer.innerText = textResponse;
+}
+
+//Functions for JSON walkthrough
```

Comment:
nit: I'd remove this comment since it isn't useful to future readers of the code